packages/hoppscotch-app/src/helpers/strategies/AxiosStrategy.ts

```diff
@@ -29,6 +29,7 @@ const getProxyPayload = (
   let payload: ProxyPayloadType = {
     ...req,
     wantsBinary: true,
+    accessToken: (!import.meta.env.VITE_PROXYSCOTCH_ACCESS_TOKEN) ? "" : import.meta.env.VITE_PROXYSCOTCH_ACCESS_TOKEN,
```

Comment:
You can use Nullish Coalescing.

```suggestion
    accessToken: import.meta.env.VITE_PROXYSCOTCH_ACCESS_TOKEN ?? ""
```

Reply:
I thought JS couldn't do this. Must be new. That's awesome! Updated!